lib/config/ChainConfigManager.ts

```diff
@@ -0,0 +1,300 @@
+import { ChainId } from '@uniswap/sdk-core';
+import { RoutingType } from '../constants';
+
+type CommonConfig = {
+  routingType: RoutingType;
+  priceImprovementBps?: number;
+  stdAuctionPeriodSecs?: number;
+  deadlineBufferSecs?: number;
+}
+
+export type DutchConfig = CommonConfig & {
```

Comment:
This naming collides with the `DutchConfig` from request https://github.com/Uniswap/unified-routing-api/blob/2554ee430c62ee452581a86ed1dc73a9274298ed/lib/entities/request/DutchRequest.ts#L12

pretty confusing, some duplication would be good